codes/quantum/qubits/homological_product.yml

```diff
@@ -7,16 +7,36 @@ code_id: homological_product
 physical: qubits
 logical: qubits
 
-name: 'Homological product code'
+name: 'Single-sector Homological product code'
 introduced: '\cite{arXiv:1311.0885}'
 
-description: 'Stub.'
+description: |
```

Comment:
STart with a sentence outlining the general properties of the code. "CSS code based on tensor product of ..."